### PR TITLE
ollama: Fix install error

### LINKED ISSUE #5908: ollama@0.1.44: decompress error

### LINKED ISSUE #5909: ollama@0.1.44: decompress error
It seems the latest version of llama is packed with inno version 6.3.0 but the innounp application installed with scoop (and indeed is what appears to be the latest version available 0.50, which looks to only be able to unpack up to version 6.1.2).

I've searched for a later version of innounp that can unpack 6.3.0 but there doesn't seem to be anything available.

Scoop error message:
"Extracting OllamaSetup.exe ... ERROR Exit code was 1!
Failed to extract files from C:\user\xxxxx\scoop\apps\ollama\0.1.44\OllamaSetup.exe."

from log file
"Signature detected: Inno Setup Setup Data (6.3.0)
This is not directly supported, but i'll try to unpack it as version 5602
; Version detected: 6300
Critical error: The setup files are corrupted. Please obtain a new copy of the program."

### LINKED ISSUE #5910: ollama@0.1.44: decompress error
```
Updating 'ollama' (0.1.43 -> 0.1.44)
Downloading new version
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: abe02b|OK  |    46MiB/s|C:/Users/otavi/scoop/cache/ollama#0.1.44#https_github.com_ollama_ollama_releases_download_v0.1.44_OllamaSetup.exe
Download: Status Legend:
Download: (OK):download completed.
Checking hash of OllamaSetup.exe ... ok.
Uninstalling 'ollama' (0.1.43)
Removing shim 'ollama.shim'.
Removing shim 'ollama.exe'.
Unlinking ~\scoop\apps\ollama\current
Installing 'ollama' (0.1.44) [64bit] from 'main' bucket
Loading OllamaSetup.exe from cache.
Extracting OllamaSetup.exe ... ERROR Exit code was 1!
Failed to extract files from C:\Users\otavi\scoop\apps\ollama\0.1.44\OllamaSetup.exe.
Log file:
  ~\scoop\apps\ollama\0.1.44\innounp.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=ollama%400.1.44%3a+decompress+error

```

### LINKED ISSUE #5911: ollama@0.1.44: decompress error
scoop update ollama fails :

```ERROR 'ollama' isn't installed correctly.
ollama: 0.1.43 -> 0.1.44
Updating one outdated app:
Updating 'ollama' (0.1.43 -> 0.1.44)
Downloading new version
Loading OllamaSetup.exe from cache.
Checking hash of OllamaSetup.exe ... ok.
Uninstalling 'ollama' (0.1.43)
Installing 'ollama' (0.1.44) [64bit] from 'main' bucket
Loading OllamaSetup.exe from cache.
Extracting OllamaSetup.exe ... ERROR Exit code was 1!
Failed to extract files from C:\Users\kamek\scoop\apps\ollama\0.1.44\OllamaSetup.exe.
Log file:
  ~\scoop\apps\ollama\0.1.44\innounp.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=ollama%400.1.44%3a+decompress+error```

### LINKED ISSUE #5912: ollama@0.1.44: decompress error
```
Installing 'ollama' (0.1.44) [64bit] from 'main' bucket
Loading OllamaSetup.exe from cache
Checking hash of OllamaSetup.exe ... ok.
Extracting OllamaSetup.exe ... ERROR Exit code was 1!
Failed to extract files from C:\Users\sharno\scoop\apps\ollama\0.1.44\OllamaSetup.exe.
Log file:
  ~\scoop\apps\ollama\0.1.44\innounp.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=ollama%400.1.44%3a+decompress+error
```

file innounp.log:
```
Signature detected: Inno Setup Setup Data (6.3.0)
This is not directly supported, but i'll try to unpack it as version 5602
; Version detected: 6300
Critical error: The setup files are corrupted. Please obtain a new copy of the program.
```

### LINKED ISSUE #5913: ollama@0.1.44: decompress error
Updating 'ollama' (0.1.41 -> 0.1.44)
Downloading new version
OllamaSetup.exe (204.9 MB) [===========================================================================================] 100%
Checking hash of OllamaSetup.exe ... ok.
Uninstalling 'ollama' (0.1.41)
Installing 'ollama' (0.1.44) [64bit] from 'main' bucket
Loading OllamaSetup.exe from cache
Extracting OllamaSetup.exe ... ERROR Exit code was 1!
Failed to extract files from F:\opt\scoop\apps\ollama\0.1.44\OllamaSetup.exe.
Log file:
  F:\opt\scoop\apps\ollama\0.1.44\innounp.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=ollama%400.1.44%3a+decompress+error

=======innounp.log-=============
Signature detected: Inno Setup Setup Data (6.3.0)
This is not directly supported, but i'll try to unpack it as version 5602
; Version detected: 6300
Critical error: The setup files are corrupted. Please obtain a new copy of the program.

### LINKED ISSUE #5915: ollama@0.1.44: decompress error

### LINKED ISSUE #5917: ollama@0.1.44: decompress error
I keep receiving this issue : 

❯ scoop install ollama
Installing 'ollama' (0.1.44) [64bit] from 'main' bucket
Loading OllamaSetup.exe from cache
Checking hash of OllamaSetup.exe ... ok.
Extracting OllamaSetup.exe ... ERROR Exit code was 1!
Failed to extract files from C:\Users\info\scoop\apps\ollama\0.1.44\OllamaSetup.exe.
Log file:
  ~\scoop\apps\ollama\0.1.44\innounp.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=ollama%400.1.44%3a+decompress+error

I am using Powershell 7.4.2 in windows 11 and I was able to run scoop install ollama in a previous version but not on 0.1.44

### LINKED ISSUE #5918: ollama@0.1.44: decompress error
Signature detected: Inno Setup Setup Data (6.3.0)                                                       This is not directly supported, but i'll try to unpack it as version 5602                               ; Version detected: 6300                                                                                Critical error: The setup files are corrupted. Please obtain a new copy of the program.

### PR DESCRIPTION
**Pull request by StarsbySea**
Closes #5908
Closes #5909
Closes #5910
Closes #5911
Closes #5912
Closes #5913
Closes #5915
Closes #5917
Closes #5918

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by StarsbySea** [comment]
@niheaven Hi, Please take a look at this PR, thanks.

**Comment by 0x00cl** [comment]
I haven't tried this but is there any reason why the `OllamaSetup.exe` was failing?
I think its something that should be looked at before changing to downloading the .zip instead.

**Comment by infomaniac50** [comment]
innounp.log says:
```
Signature detected: Inno Setup Setup Data (6.3.0)
This is not directly supported, but i'll try to unpack it as version 5602
; Version detected: 6300
Critical error: The setup files are corrupted. Please obtain a new copy of the program.
```

**Comment by infomaniac50** [comment]
The OllamaSetup.exe installer works fine by itself. It drops the files in the users AppData folder as a non-admin install. The tray app and the server run fine.

Ollama v0.1.41 uses Inno Setup v6.1.0 where as Ollama v0.1.44 uses Inno Setup v6.3.0.

I assume the log is right and the new Inno Setup installer is not supported by innounp.

**Comment by infomaniac50** [comment]
See Also:

- https://jrsoftware.org/isinfo.php
- https://innounp.sourceforge.net/

**Comment by 0x00cl** [comment]
Scoop uses [innounp](https://innounp.sourceforge.net/), latest version is 0.50 and supports upto Inno 6.1.2. which was last updated  2020.12.18 and latest Inno version is 6.3.1 (2024-06-10).

> ... But since it is not updated in a timely fashion, and so does not support the latest IS most of the time, this program was born.

A bit ironic :p

**Comment by 0x00cl** [comment]
Looks like [innoextract](https://github.com/dscharrer/innoextract) could be an alternative? It already has a merge request to support 6.3.1 Inno Setup

**Review by sharno — APPROVED** [review]

File: bucket/ollama.json [hunk 1]
```diff
@@ -3,23 +3,14 @@
     "description": "Get up and running with large language models locally.",
     "homepage": "https://ollama.com/",
     "license": "MIT",
-    "url": "https://github.com/ollama/ollama/releases/download/v0.1.44/OllamaSetup.exe",
-    "hash": "640c587550810b941e384618e4a0130e3cda56ac67f7894b821c141aefd6ad0c",
-    "innosetup": true,
+    "url": "https://github.com/ollama/ollama/releases/download/v0.1.44/ollama-windows-amd64.zip",
+    "hash": "483fef57f094f5a52d865182a0d6f9a2e8a59eb44dd756b4517ea9b747d90338",
```
Comment (wickles) [inline bucket/ollama.json:7]: Needs rebase and update for version `0.1.47`.

**Pull request closed without merging**